homeassistant/components/matter/manifest.json

```diff
@@ -6,5 +6,5 @@
   "dependencies": ["websocket_api"],
   "documentation": "https://www.home-assistant.io/integrations/matter",
   "iot_class": "local_push",
-  "requirements": ["python-matter-server==3.6.0"]
+  "requirements": ["python-matter-server==3.6.3"]
```

Comment:
We made a breaking change in 3.6.2:
https://github.com/home-assistant-libs/python-matter-server/pull/339

Reply:
yeah, unfortunately you are right. its just a rename but breaking. I'va updated the code with the rename of subscribe to subscribe_events